Notifier-service/src/services/conference.spec.ts

```diff
@@ -1,16 +1,24 @@
 import { ConferenceModel } from '../interfaces/models/conference'
 import { ConferenceServiceI } from './conference'
+import * as dotenv from 'dotenv'
+import { MongoDb } from '../database/mongodb'
+import { ConferenceModelMongo } from '../models/conference'
+import { ConferenceDocument } from '../schemas/conferences'
 
 describe("Testing Conferences Service Implementation " ,() => {
+    dotenv.config()
     let ModelMock = jest.fn<ConferenceModel , []>()
     let model = new ModelMock()
     let categories = ["category1" , "category2"]
     model.getCategories = jest.fn(() => Promise.resolve(["category1" , "category2"]))
     model.getConferences = jest.fn( () => Promise.resolve([]) )
+    const mongo = new MongoDb()
```

Comment:
* Mock this also ....

Reply:
oh ok. I had not done that as I would have had to mock confModelMongo as well. Could you suggest me how to handle this. If I mock confModelMongo entirely then the actual working of the getOne method won't be tested as it queries a database. 
Instead,should I mock this here and create a test file for the methods of conferenceModelMongo class?

Reply:
* For now , just ignore the test case. And test for the function existence. I will later create issue for creating test specific to this. With proper dev vs prod division to make sure it plays with other environments also. Plus we can also extract the class object from the inversify DI container. So for now dont bother with the test. Write a simple one testing the existence  of the function.

Reply:
Sure, I had done that and pushed my code, you can see the latest version which only tests this function